mlir/utils/jenkins/Jenkinsfile.downstream

```diff
@@ -41,8 +43,27 @@ pipeline {
                         -DMLIR_ROCM_RUNNER_ENABLED=1 \
                         -DMLIR_MIOPEN_DRIVER_ENABLED=1
 
-                    # build LLVM / MLR and run tests
-                    cmake --build . --target check-mlir
+	     	    # premerge checks
+                    cd ..
+                    if [ ! -f ./compile_commands.json ]; then
+                       ln -s build/compile_commands.json compile_commands.json
+                    fi
+		    python3 ./mlir/utils/jenkins/static-checks/premerge-checks.py
+
+		    if [ $? != 0 ]; then
+			exit 1
+		    fi
```

Comment:
I think Jenkins does this by default, does it?

Reply:
I think by default Jenkins will exit the current stage and continue to the next stage.

Reply:
That's probably not true. From my experience with MIOpen CI, they always stop when one stage has a failure, yet their [Jenkinsfile](https://github.com/ROCmSoftwarePlatform/MIOpen/blob/develop/Jenkinsfile) does not come with any `exit 1`.

Reply:
Their code catches exceptions. Maybe that is how it exits.

Reply:
Do you know whom in MIOpen group I could contact with to ask this question?

Reply:
@daniellowell is the expert in this. Is this explicit `exit 1` necessary in the pipeline for jenkins to abort execution?

Reply:
@yiqian1  Now instead of waiting for the reply, why not push for an experimental commit and see if Jenkins behaves as expected? You can easily `reset HEAD` to original once you figured it out.